codice/ddf
# DDF-2507 CSW matchcase will now be ignored for non-string queries

#### What does this PR do?
CSW numerical queries were getting no results due to being rejected thanks to a default matchcase of "false". Now numerical queries will default to true.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@brendan-hofmann 
@harrison-tarr 
@jrnorth 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl
@kcwire

#### How should this be tested?
Full build and ingest a metacard that will have a numerical attribute. Confirm that a refined search over a CSW endpoint on that numerical attribute gets results.

#### Checklist:
